tests/testthat/test-Data-methods.R

```diff
@@ -656,3 +656,17 @@ test_that("tidy-DataGeneral creates the correct tibble", {
   expected$Tox <- c(FALSE, TRUE, FALSE)
   expect_equal(tidy(d), expected)
 })
+
+# See https://github.com/openpharma/crmPack/issues/866
```

Comment:
```suggestion
```
we don't do that, note that via GitHub blame we can find out later where which test was added/edited